server/chat.ts

```diff
@@ -263,7 +263,21 @@ export class PageContext extends MessageContext {
 		parts.shift();
 		while (handler) {
 			if (typeof handler === 'function') {
-				let res = await handler.bind(this)(parts, this.user, this.connection);
+				let res;
+				try {
+					res = await handler.bind(this)(parts, this.user, this.connection);
```

Comment:
Looks like Ashe forgot about `call` existing.

```suggestion
					res = await handler.call(this, parts, this.user, this.connection);
```